OpenRA.Mods.Common/Commands/DebugVisualizationCommands.cs

```diff
@@ -57,6 +63,11 @@ public void InvokeCommand(string name, string arg)
 					debugVis.RenderGeometry ^= true;
 					break;
 
+				case "showscreenmap":
+					if (devMode == null || devMode.Enabled)
```

Comment:
Why add the extra dependency to make a special snowflake out of this when none of the others are doing it?

Reply:
Because this one displays the positions of every actor on the map, ignoring shroud/fog/cloak/etc. The others can't be used maliciously to cheat.

Reply:
Thought so, so I checked the rest. The `CombatDebugOverlay` also works through for, but not shroud (kinda odd). Maybe we should fix that?
Also if this is intended to *not* let me enable the overlay through the chat command without turning on "cheats", then I don't think it worked.  :/

Reply:
Cheats are force enabled on any game with only one real person.